### PR TITLE
new timeblock entry

### PR DESCRIPTION
**Pull request by jasmine2000**
only basic functionality for now! styling to come 😄

### CODE REVIEW

File: client/components/Profile/CalendarComponent.vue [hunk 1]
```diff
@@ -293,49 +304,48 @@ export default {
               this.hasRequestable = true;
             }
         },
-        showDatePicker() {
+        showTimeSelector() {
```
Comment (tejalreddy) [inline client/components/Profile/CalendarComponent.vue:307]: Have a check that sends an error/says something if a user tries to add a time between days (ex: 11:30 pm - 12:30 am). Right now, it does nothing

File: client/components/Profile/CalendarComponent.vue [hunk 2]
```diff
@@ -457,39 +467,22 @@ export default {
...
 
 .pastDay {
-  width: 100%;
-  border: 1px solid black;
-  min-height: 100px;
   background-color:lightgray;
 }
 
-.today {
+.day {
```
Comment (tejalreddy) [inline client/components/Profile/CalendarComponent.vue:473]: How would this look like with a scrollbox in each day? Could be helpful to make the calendar pretty uniform no matter how many entries there are. Should just be adding a overflow-y: scroll to the css

**Review by tejalreddy — CHANGES_REQUESTED** [review]

File: client/components/Profile/CalendarComponent.vue (hunk 1 above)
Comment (jasmine2000) [reply client/components/Profile/CalendarComponent.vue:307]: wait how did you do that? you can only select one day at a time right

Comment (jasmine2000) [reply client/components/Profile/CalendarComponent.vue:307]: oh! i see whats happening - if the end is before the start nothing happens. i'll add the middleware

File: client/components/Profile/CalendarComponent.vue (hunk 2 above)
Comment (jasmine2000) [reply client/components/Profile/CalendarComponent.vue:473]: it looks wack rn but ill leave the code and we can look tm

**Pull request merged**